libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFAuthenticationManager.m

```diff
@@ -1416,8 +1435,14 @@ - (void)oauthCoordinator:(SFOAuthCoordinator *)coordinator displayConfirmationMe
                                                           }];
     [alert addAction:cancelAction];
 
-    [[SFRootViewManager sharedManager] pushViewController:alert];
-}
+    SFSDKWindowManager.sharedManager.authWindow.viewController = [self blankViewController];
+    [SFSDKWindowManager.sharedManager.authWindow enable:NO withCompletion:^{
+        [SFSDKWindowManager.sharedManager.authWindow.viewController presentViewController:alert animated:NO completion:^{
```

Comment:
It looks like this completion block is empty. Can we replace it with `nil` instead?